app/assets/stylesheets/components/_language_selector.scss

```diff
@@ -0,0 +1,16 @@
+.language-selector__select {
+  margin-bottom: $baseline-unit*2;
+}
+
+.language-selector__delete {
+  margin-left: 0.5em;
+  cursor: pointer;
```

Comment:
Have made the change. There was a class — `button-link`. Not sure of its providence exactly.